packages/ckeditor5-bookmark/theme/bookmark.css

```diff
@@ -35,5 +35,11 @@
 		&.ck-widget_selected:hover {
 			outline: none;
 		}
+
+		& .ck-bookmark__icon {
+			& .ck-icon {
+				width: 1em;	
```

Comment:
We need to define font size of the icon, but it should also look good in the Headings.

With that style rule:
<img width="192" alt="image" src="https://github.com/user-attachments/assets/5cfea97e-9174-465b-9d9c-a9573833f912">

Without that style rule:
<img width="206" alt="image" src="https://github.com/user-attachments/assets/58bf97ce-e9b1-4fde-8b28-b514912ad237">

Reply:
I feel solution proposed by @pszczesniak is better since the em unit is related to the text height, not width.

Reply:
BTW, when we finish this iteration we need a review from @dwandeljanicka. Maybe some sizes would needs to be updated.

Reply:
I agree, I am just worried about the value of `em`, do you think 0.75 is not too small? I feel like it harder to hit into the icon because of its small size.

Reply:
Please, suggest the size that would suite best from your point of view, TBH don't know, need to run manual test and check it out.